interview-questions.md

```diff
@@ -8,50 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Interview questions are your opportunity to share your prospective. The interviewers are not looking for you to read verbatim what a webpage states. They want to know how you comprehend what you have read. Please go through each interview question and write your research in your own words. 
Additionally, as a reviewer I want to support your progress. I cannot do that unless you share your thoughts.